e2e/cypress/integration/enterprise/guest_accounts/guest_identification_ui_spec.js

```diff
@@ -226,7 +217,7 @@ describe('MM-18045 Verify Guest User Identification in different screens', () =>
 
     it('Verify Guest Badge not displayed in Search Autocomplete', () => {
         // # Search for the Guest User
-        cy.get('#searchBox').type('from:user');
+        cy.get('#searchBox').type('from:');
```

Comment:
Yes, same. Previously, there was too many users which end up needing to type "user" prefix to show some of them including the newly created guest account. But now, guest account has default prefix of "guest", so I removed the keyword so that all 3 users in this team will show up in the suggestion list.